README.md

```diff
@@ -1,5 +1,52 @@
-Chitter Challenge
-=================
+# Will's README
```

Comment:
Lovely work!
- Nicely written README
- The code is very easy to read, good use of the SRP in your methods and classes 
- Couldn't find anything that I would've done differently